py/kubeflow/kfctl/testing/util/kfctl_go_test_utils.py

```diff
@@ -285,6 +289,16 @@ def kfctl_deploy_kubeflow(app_path, project, use_basic_auth, use_istio, config_p
     apply_kubeflow(kfctl_path, app_path)
   return app_path
 
+def push_kfctl_to_gcs(app_path):
+  bucket = "kubernetes-jenkins"
```

Comment:
Why are you hardcoding the bucket here? How are we doing it for other artifacts?

Reply:
I was following the same logic in the function to see if I can write to the bucket ok. I'll switch it out with these [two functions](https://github.com/kubeflow/testing/blob/master/py/kubeflow/testing/util.py#L845).

Reply:
nit: Maybe use the constant defined here: https://github.com/kubeflow/testing/blob/25b87a36f03c207f69abbc9e18500e9ac75647d0/py/kubeflow/testing/prow_artifacts.py#L19

Reply:
Done